src/user-guide/writing-workflows/jinja2.rst

```diff
@@ -7,44 +7,36 @@ Jinja2
 .. tutorial::
    Configuration Consolidation Tutorial <tutorial-cylc-consolidating-configuration>
 
-.. note::
+Cylc supports use of the `Jinja2`_ template processor in workflow
+configurations. Jinja2 variables, expressions, loop control structures,
+conditional logic, etc., are processed to generate the final configuration. To
+the template processor, Jinja2 code is embedded in arbitrary text, but the
+result after processing must be valid Cylc syntax.
 
-   This section needs to be revised - the Parameterized Task feature
-   introduced in cylc-6.11.0 (see :ref:`User Guide Param`) provides
-   a cleaner way to auto-generate tasks without coding messy Jinja2 loops.
+Use of Jinja2 should be declared with a hash-bang comment in the first line of
+the :cylc:conf:`flow.cylc` file:
```

Comment:
The shebang is now a couple of paragraphs down from here instead of immediately after

Reply:
Fixed.